node/neighbor.go

```diff
@@ -424,3 +425,85 @@ func (localNode *LocalNode) maybeAddRemoteNode(remoteNode *nnetnode.RemoteNode)
 
 	return nil
 }
+
+func (localNode *LocalNode) VerifySigChain(sc *pb.SigChain, height uint32) error {
+	c, ok := localNode.nnet.Network.(*chord.Chord)
+	if !ok {
+		log.Fatal("Overlay is not chord")
+	}
+
+	if config.SigChainVerifySkipNode.GetValueAtHeight(height) {
+		// only needs to verify node to node hop, and no need to check last node to
+		// node hop because it could be successor
+		for i := 1; i < sc.Length()-3; i++ {
```

Comment:
Suggest `for i, elem := range sc[1:sc.Length()-2]` for more legible.

Reply:
Better not to use it because sc[1:sc.Length()-2] will create a new byte slice. Also the `i` will be different in original sigchain, making code harder to read